server/middleware/authorisationForUrlMiddleware.ts

```diff
@@ -0,0 +1,14 @@
+import { RequestHandler } from 'express'
+
+import logger from '../../logger'
+import Role from '../authentication/role'
+
+export default function authorisationForUrlMiddleware(authorisedRoles: Role[] = []): RequestHandler {
+  return (req, res, next) => {
+    if (authorisedRoles.length && !res.locals.user.roles.some((role: Role) => authorisedRoles.includes(role))) {
```

Comment:
**authorisedRoles.length** says if no roles are passed, allow anyone to see the route. This is the desired behaviour